tests/cases/standalone/common/catalog/schema.result

```diff
@@ -19,6 +19,14 @@ SHOW DATABASES LIKE '%public%';
 | test_public_schema |
 +--------------------+
 
+SHOW DATABASES WHERE Schemas='test_public_schema';
```

Comment:
Could you please also add a case for `like`?

Reply:
I guess that we've already add it to this file:

https://github.com/GreptimeTeam/greptimedb/blob/8bea85395444e640cfef93cc38ff6445213677c8/tests/cases/standalone/common/catalog/schema.sql#L7